components/header/header.module.scss

```diff
@@ -0,0 +1,122 @@
+@import '../../styles/variables';
+@import '../../styles/mixins';
+
+.container {
+  background: $color-dark;
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  z-index: 99;
+}
+
+.header {
+  background: $color-dark;
+  display: flex;
+  height: $header-height;
+  justify-content: space-between;
+  margin: auto;
+  max-width: $max-content;
+  position: sticky;
+
+  @include breakpoint(medium-only) {
+    padding: 0.5em;
+  }
+
+  @include breakpoint(small) {
+    padding: 0.5em;
+  }
+}
+
+.logo {
+  display: flex;
+  margin: 0.5em;
+
+  @include breakpoint(medium-only) {
+    margin-top: 0em;
+    margin-left: 0.5em;
+  }
+
+  @include breakpoint(small) {
+    margin-top: 0em;
+    margin-left: 0.5em;
+  }
+}
+
+.logoText {
+  font-size: 1.25em;
+  font-weight: 400;
+  margin: 0.5em;
+}
+
+.menu {
+  align-items: center;
+  background: $color-dark;
+  display: flex;
+  justify-content: center;
+  opacity: 0.9;
+
+  @include breakpoint(small) {
+    -webkit-transition: all 0.3s ease;
+    flex-direction: column;
+    height: 100%;
+    position: fixed;
+    right: 0;
+    top: -1000px;
```

Comment:
Let's change this to `rem` for consistency with all other units in this style document

Reply:
not sure if it's just on my side (sometimes the gitUI does that), but I am seeing this as `px`?